docs/en/Getting-Started-on-Cluster.md

```diff
@@ -0,0 +1,334 @@
+---
+layout: global
+title: Getting Started on a Cluster
+group: User Guide
+priority: 2
+---
+
+* Table of Contents
+{:toc}
+
+The [Quick Start Guide](Getting-Started.html) shows how to install Alluxio locally on a single machine.
+In this advanced quick start guide, we will install Alluxio on a 3-node Linux cluster, configure a
+AWS S3 bucket as UFS (Under File System), and perform basic tasks with the data in Alluxio.
+
+During this guide, you will:
+
+* Download and configure Alluxio in a Linux cluster
+* Configuration for AWS
+* Validate Alluxio environment
+* Set up a distributed storage as UFS
+* Start Alluxio on multiple nodes
+* Perform basic tasks via Alluxio Shell
+* **[Bonus]** Run sample Hadoop MapReduce job in Alluxio cluster
+* **[Bonus]** Run sample Apache Spark job in Alluxio cluster
+* Shutdown Alluxio
+
+**[Required]** In this documentation, we will use AWS S3 as the example distributed storage as Alluxio under file system.
+Please make sure you have an [AWS account with an access key id and secret access key](http://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSGettingStartedGuide/AWSCredentials.html),
```

Comment:
The link is for SQS. Is there a more general AWS link to point to?

Reply:
done, thx